docs/developer_guide/getting_started.md

```diff
@@ -9,27 +11,25 @@ In addition to traditional RPC wallet communication, monero-javascript can manag
 ![Monero-javascript hierarchy](../architecture.png?raw=true)*Monero-javascript manipulates XMR and communicates with the blockchain via three channels: RPC wallet servers, RPC daemon servers, and Wasm wallets.*  
 
 ## Initial Setup
+
 ### Install Node.js and the Node package manager (npm)
+
 Make sure you have Node.js and npm installed; you need them to obtain and use the monero-javascript library. See the ["Node.js and npm"](https://github.com/monero-ecosystem/monero-javascript/blob/master/docs/developer_guide/installing_prerequisite_software.md#nodejs-and-npm) section of the prerequisite installation guide for step-by-step instructions for downloading and installing Node.js and npm.
 
 ### Create a new Node.js project
-1. Create and enter a new directory to hold the project:
-  ### Linux/Mac
-  1. `mkdir ~/offline_wallet_generator`
-  2. `cd ~/offline_wallet_generator`
-
-  ### Windows
-  1. `mkdir C:\offline_wallet_generator`
-  2. `cd C:\offline_wallet_generator`
 
+1. Create and enter a new directory to hold the project:
+  ```
+  mkdir ~/offline_wallet_generator`
+  b. `cd ~/offline_wallet_generator`
+  ```
```

Comment:
I need to fix this to read:

1. Create and enter a new directory to hold the project:
  ```
  mkdir ~/offline_wallet_generator
  cd ~/offline_wallet_generator
  ```